core/src/main/kotlin/riven/core/service/entity/EntityService.kt

```
@@ -0,0 +1,217 @@
+            )
+        )
+    }
+
+    /**
+     * Archive or restore an entity.
+     * An archival operation will move the entity to a separate archival table, and a restoration will bring it back.
+     */
+    @Transactional
+    @PreAuthorize("@organisationSecurity.hasOrg(#id)")
+    fun archiveEntity(id: UUID, archive: Boolean): Entity {
+        TODO()
```

detekt (1.23.8)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)